src/te/operation/create_primfunc.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef TVM_TE_OPERATION_CREATE_PRIMFUNC_H_
+#define TVM_TE_OPERATION_CREATE_PRIMFUNC_H_
+
+#include <tvm/runtime/container/array.h>
+#include <tvm/te/tensor.h>
+#include <tvm/tir/function.h>
+
+namespace tvm {
+namespace tir {
+
+PrimFunc CreatePrimFuncFromOutputs(const Array<te::Tensor>& outputs);
```

Comment:
Let's expose two APIs in `include/tvm/te/operation.h`:
- CreatePrimFunc
- CreatePrimFuncFromOutputs

Reply:
`include/tvm/te/operation.h` defines things under `tvm::te` namespace, while `create_primfunc.cc` defines those functions under `tvm::tir`.

There is also `include/tvm/tir/op.h`, but things defined there are of different nature than `CreatePrimFunc`.

For now, I exposed them under `create_primfunc.h`.

Reply:
Thanks for sharing the consideration! It definitely makes sense to me